sleap_io/io/nwb.py

```diff
@@ -162,20 +197,27 @@ def append_labels_data_to_nwb(
             processing_module_name, nwbfile
         )
 
+        # Propagate video metadata
+        default_metadata["original_videos"] = [f"{video.filename}"]  # type: ignore
+        default_metadata["labeled_videos"] = [f"{video.filename}"]  # type: ignore
+
```

Comment:
Was getting error `sleap_io/io/nwb.py:201: error: Incompatible types in assignment (expression has type "List[str]", target has type "str")` in liens 200 and 201. Is there are more elegant way of avoiding this?

Reply:
Ugh, yeah, the typing for complicated `dict` value types is a bit hairy. Technically I suppose this is intended to discourage the use of `dict`s as generic grab bags of different data structures that aren't documented explicitly, but it sacrifices a lot of flexibility.

Ignoring is fine for now. In my view, typing is a "best effort" practice and should be ignored if it makes the code harder to read or iterate on.